app/javascript/components/enrollment/steps/CaseInformation.js

```diff
@@ -0,0 +1,401 @@
+import React from 'react';
+import { PropTypes } from 'prop-types';
+import { Alert, Button, Card, Col, Form } from 'react-bootstrap';
+import ReactTooltip from 'react-tooltip';
+import * as yup from 'yup';
+import axios from 'axios';
+import moment from 'moment';
+import _ from 'lodash';
+
+import confirmDialog from '../../util/ConfirmDialog';
+import DateInput from '../../util/DateInput';
+import FirstPositiveLaboratory from '../../patient/laboratory/FirstPositiveLaboratory';
+import PublicHealthManagement from './PublicHealthManagement';
+
+class CaseInformation extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      ...this.props,
+      current: this.props.currentState,
+      errors: {},
+      modified: {},
+      sorted_jurisdiction_paths: _.values(this.props.jurisdiction_paths).sort((a, b) => a.localeCompare(b)),
+      jurisdiction_path: this.props.jurisdiction_paths[this.props.currentState.patient.jurisdiction_id],
+      originalJurisdictionId: this.props.currentState.patient.jurisdiction_id,
+      originalAssignedUser: this.props.currentState.patient.assigned_user,
+      assigned_users: this.props.assigned_users,
+      selected_jurisdiction: this.props.selected_jurisdiction,
+      isEditMode: window.location.href.includes('edit'),
```

Comment:
same here

Reply:
fixed in https://github.com/SaraAlert/SaraAlert/pull/975/commits/441588e6694ae61c4bd1da5b7da3be4de8fd5aa9